### PR TITLE
You can change your profile picture!

### PR DESCRIPTION
**Pull request by citelao**
Waiting for update to Hull.js before push.

### CODE REVIEW

**Comment by oliger** [comment]
Hey can you merde `develop` please :)

**Comment by citelao** [comment]
`Hull.config().services.storage.hull` is undefined... did the API change?

**Comment by unity** [comment]
This has to be changed by @oliger or @sbellity - it should be enabled automatically. We decided that a while back. I'm not sure it's been done

**Comment by oliger** [comment]
Not changed by me. @sbelity ?

**Comment by citelao** [comment]
It used to work. I think it might be assigned wrong in a newer version.

**Comment by sbellity** [comment]
@citelao it probably worked before on an organization that had the hull_store service enabled. 
We could either enable the feature for all orgs and/or add an entry in the manifest to specify which services need to be installed for the ship to work.

**Comment by citelao** [comment]
@unity Does it have to be manually enabled? I don't remember turning it on myself.

**Comment by unity** [comment]
Enable for all orgs

**Comment by unity** [comment]
One less thing to worry abkht

**Comment by citelao** [comment]
@unity @sbellity It seems to be resolved.

**Comment by citelao** [comment]
Ready to merge?

File: src/lib/engine.js [hunk 1]
```diff
@@ -378,6 +382,51 @@ assign(Engine.prototype, EventEmitter.prototype, {
...
 
+  updatePicture(file) {
+    return new Promise((resolve, reject) => {
+      let s3config = Hull.config().services.storage.hull;
+
+      let url = s3config.url;
+      let type = file.type;
+      let name = file.name;
+
+      let form = new FormData();
```
Comment (sbellity) [inline src/lib/engine.js]: FormData is only supported on IE10+ 

http://caniuse.com/#feat=xhr2
https://developer.mozilla.org/en-US/docs/Web/API/FormData

Comment (citelao) [inline src/lib/engine.js]: gahhhhhhhhhhhhhhhhhhhhhhhhhhh

I'm not smart enough to solve this. Apparently [you can't](http://stackoverflow.com/a/13497736/788168)?

Comment (citelao) [inline src/lib/engine.js]: https://github.com/Modernizr/Modernizr/wiki/HTML5-Cross-Browser-Polyfills#file-api?

Comment (citelao) [inline src/lib/engine.js]: @sbellity What's your recommendation?

Comment (citelao) [inline src/lib/engine.js]: yeah we should merge this sometime @oliger @sbellity @unity, help?

**Pull request closed without merging**